web/src/constants.js

```diff
@@ -1,2 +1,2 @@
 export const TOKEN_KEY = "token";
-export const BASE_URL = "https://around-311020.wl.r.appspot.com";
+export const BASE_URL = "http://auth.api.staging.allgame.fun";
```

Comment:
Please add them through environmental variables so that they have different values for local, staging and prod.